src/components/POLine/Eresources/EresourcesForm.js

```diff
@@ -88,13 +89,9 @@ class EresourcesForm extends Component {
           />
         </Col>
         <Col xs={6} md={3}>
-          <Field
-            className={css.checkboxValignCenter}
-            component={Checkbox}
-            fullWidth
-            label={<FormattedMessage id="ui-orders.eresource.createItem" />}
-            name="eresource.createInventory"
-            type="checkbox"
+          <InventoryRecordTypeSelectField
+            label="ui-orders.eresource.createItem"
+            name="eresource.createItem"
```

Comment:
are you sure about `createItem`? in view component you're looking for `createInventory`

Reply:
You're right. Checked in the schema